.github/workflows/ci.yml

```diff
@@ -392,17 +302,26 @@ jobs:
           flutter drive --target=test_driver/app.dart --dart-define=testName="" --suppress-analytics
         working-directory: ./flutter/realm_flutter/tests
 
+      - name: Cleanup apps
+        if: always()
+        run: dart run realm_dart delete-apps --appIds '${{ steps.deploy-apps.outputs.appIds }}' --baas-url ${{ secrets.REALM_QA_URL }} --atlas-cluster ${{ needs.deploy-cluster.outputs.clusterName }} --api-key ${{ secrets.ATLAS_QA_PUBLIC_API_KEY }} --private-api-key ${{ secrets.ATLAS_QA_PRIVATE_API_KEY }} --project-id ${{ secrets.ATLAS_QA_PROJECT_ID }} --differentiator '${{ env.BAAS_DIFFERENTIATOR }}'
+
+# Android jobs
+
   flutter-android:
-    runs-on: macos-latest
+    runs-on: ubuntu-latest
```

Comment:
We can't run the android emulator on ubuntu - it's insanely slow.

Reply:
Done